### PR TITLE
D758967: add domain validity and change a class name

### PR DESCRIPTION
**Pull request by apollo-crum**

### CODE REVIEW

File: pyaedt/emit.py [hunk 1]
```diff
@@ -247,6 +247,20 @@ def set_max_simultaneous_interferers(self, val):
         eng = self.emit_obj._emit_api.get_engine()
         eng.max_simultaneous_interferers = val
 
+    @pyaedt_function_handler()
+    def is_domain_valid(self, ret_val, domain):
+
```
Comment (MaxJPRey) [inline pyaedt/emit.py:252]:
```suggestion
```

File: pyaedt/emit.py [hunk 2]
```diff
@@ -247,6 +247,20 @@ def set_max_simultaneous_interferers(self, val):
...
+        """
+        Return true if the given domain is valid for the current Revision
```
Comment (MaxJPRey) [inline pyaedt/emit.py:254]:
```suggestion
        Return ``True`` if the given domain is valid for the current revision.
```

File: pyaedt/emit.py [hunk 3]
```diff
@@ -247,6 +247,20 @@ def set_max_simultaneous_interferers(self, val):
...
+
+        Examples
+        ----------
+        >>> domain = aedtapp.interaction_domain()
+        >>> if aedtapp.results.is_domain_valid(domain):
```
Comment (MaxJPRey) [inline pyaedt/emit.py:259]: This way we can also see the returned value.
```suggestion
        >>> aedtapp.results.is_domain_valid(domain)
        True
```

File: pyaedt/emit.py [hunk 4]
```diff
@@ -247,6 +247,20 @@ def set_max_simultaneous_interferers(self, val):
...
+        """
+        eng = self.emit_obj._emit_api.get_engine()
+        return eng.is_domain_valid(domain)
```
Comment (MaxJPRey) [inline pyaedt/emit.py:262]:
```suggestion
        return engine.is_domain_valid(domain)
```

**Review by myoung301 — APPROVED** [review]

**Pull request merged**